packages/test/test-drivers/src/routerliciousTestDriver.ts

```diff
@@ -111,8 +111,10 @@ export class RouterliciousTestDriver implements ITestDriver {
     ) {
     }
 
-    async createContainerUrl(testId: string): Promise<string> {
-        return `${this.serviceEndpoints.hostUrl}/${encodeURIComponent(this.tenantId)}/${encodeURIComponent(testId)}`;
+    async createContainerUrl(testId: string, containerUrl?: IResolvedUrl): Promise<string> {
```

Comment:
@alpaix i'm seeing failures in some end to end test i added. they only fail on tinylicious. my hunch is that the failures are related to these changes as well, but it's not clear to me how to fix them. I have a feeling the optional parameter here is not really optional in all cases

 can you take a look at https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-end-to-end-tests/src/test/noDetlaStream.spec.ts and their tinylicious failures in the lab, and let me know how to fixe them.

if this is the cause, it think there is more work here to make the api more usable.

Reply:
I've updated the `noDeltaStream.spec.ts` test suite in this PR to consume the new parameter and verified all tests passed on a test branch. I tried to avoid invasive breaking API changes in the test infra, rather addressed specific test failures. Does it make sense?
![image](https://user-images.githubusercontent.com/8239356/140563500-d9b0a4c4-c62d-4352-8c58-b4e0dde28e86.png)

Reply:
I'll check whether there are no more calls to `createContainerUrl` left untouched. Will make the parameter required if so.

Reply:
We have a few other occurrences of this API calls in other places where the `documentId` aka `testId` is properly used and points to the actual container ID. I do agree we should normalize the usage pattern across all tests to avoid confusion. That'll be out of scope of this PR though. I'll make a separate iteration over it later. I'll merge the PR if there are no objections.

Reply:
sound perfect. end fine just opening an issue. thanks for taking care of it

Reply:
Filed #8156